sql/_05_plcsql/_01_testspec/_02_declaration/_07_local_proc/_05_common/cases/02_07_05-03_normal_return_with_value.sql

```diff
@@ -0,0 +1,16 @@
+--+ server-message on
+
+-- error: local procedure returns with a value
+
```

Comment:
The file name must be changed as shown below.

02_07_05-03_error_return_with_value.sql

Reply:
Yes. Thank you